core/services/ocr2/plugins/functions/config/config_types.proto

```diff
@@ -9,11 +9,26 @@ enum AggregationMethod {
     AGGREGATION_MEDIAN = 1;
 }
 
+message OracleIDtoKeyShareIndex {
```

Comment:
Can you add a comment explaining the meaning of this? It's not attached to any other protos. Does it need to be a proto in the first pleace?

Reply:
This is copied from the tdh2 decryption plugin, but yes I think it can be removed here.